bin/start-containter-with

```diff
@@ -2,15 +2,15 @@
 
 # docker image import https://tool.transitionmonitor.com/downloads/2diirunner_1.3.0.tar.xz 2diirunner:1.3.0
 
-analysis=$(pwd)
-data=$(dirname $(pwd))/pacta-data
-report=$(dirname $(pwd))/create_interactive_report
-stresstest=$(dirname $(pwd))/StressTestingModelDev
+analysis="$(pwd)"
+data="$(dirname $(pwd))/pacta-data"
+report="$(dirname $(pwd))/create_interactive_report"
+stresstest="$(dirname $(pwd))/StressTestingModelDev"
 
 docker run --rm -ti \
   --mount type=bind,source=$analysis,target=/bound \
   --mount type=bind,source=$data,target=/pacta-data \
   --mount type=bind,source=$report,target=/create_interactive_report \
   --mount type=bind,source=$stresstest,target=/StressTestingModelDev \
```

Comment:
these lines should also quote the shell variable strings…
```
  --mount type=bind,source="$analysis",target=/bound \
  --mount type=bind,source="$data,target=/pacta-data \
  --mount type=bind,source="$report",target=/create_interactive_report \
  --mount type=bind,source="$stresstest",target=/StressTestingModelDev \
```

Reply:
Oops, thanks!